shadow-cljs.edn

```diff
@@ -71,6 +71,12 @@
          :closure-defines {frontend.util/NODETEST true}
          :devtools        {:enabled false}}
 
+  :test-auto {:target          :node-test
+              :output-to       "static/tests.js"
+              :closure-defines {frontend.util/NODETEST true}
+              :devtools        {:enabled false}
+              :autorun true}
```

Comment:
This can be merged to the above build.
```clojure
:autorun false
:dev { :autorun true}
```

Reply:
I looked into this but it didn't work. Most of the tests fail with `clojure -M:test release test`. I did find a way to remove this config thanks to `--config-merge` so I can remove this